canonicalwebteam/blog/djangoblog/apps.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
This is used if we want python 2 support. Is it something we need? (@nottrobin )

Reply:
I forgot we already use f string. So Python 3

Please remove those lines

Reply:
Updated :)